examples/eia/docs/components/charts.js

```diff
@@ -110,9 +111,14 @@ export function countryInterchangeChart(width, height, usDemandGenForecast, coun
         y: (d) => d.value / 1000,
         curve: "step",
         fill: "name",
-        tip: true
-      }),
-      Plot.ruleY([0], {strokeOpacity: 0.3})
+        tip: true,
+        title: (d) =>
+          `Country: ${d.name}\nDate: ${d3.timeFormat("%-d %b")(d.date)} ${d.date.toLocaleString("en-US", {
+            hour: "numeric",
+            hour12: true
+          })}\nExported: ${d.value / 1000} GWh`
```

Comment:
Feels a little funny to mix-and-match d3-time-format and date.toLocaleString here. Also, the `hour12: true` should be implied for the en-US locale. Also, we probably want to use number.toLocaleString to display the number, too, in case there are thousands.

```suggestion
          `Country: ${d.name}\nDate: ${d.date.toLocaleString("en-US", {
            month: "short",
            day: "numeric",
            hour: "numeric"
          })}\nExported: ${(d.value / 1000).toLocaleString("en-US")} GWh`
```

It’s not exactly the same date format (`Apr 10, 9 AM` instead of `10 Apr 9 AM`), but I think the new format is also more common and readable.